prompts/all.json

```diff
@@ -81,17 +81,17 @@
     },
     {
       "id": "customer-support-banking",
-      "label": "Generate realistic customer support questions for an online banking system",
+      "label": "Customer support questions for a bank",
       "value": "Create a mock dataset: Columns: id, customer_support_question. Generate realistic customer support questions for an online banking system. Generate 30 rows of data."
     },
     {
       "id": "product-reviews",
-      "label": "Generate positive and negative reviews for a given list of products",
+      "label": "Positive and negative product reviews",
       "value": "Generate positive and negative reviews for common household products purchased online. Create 25 rows of data. Columns are: the product name, number of stars (1-5), review and customer id"
     },
     {
       "id": "texas-professionals",
-      "label": "Generate a mock dataset for professionals based in Austin, Texas",
+      "label": "Dataset of professionals based in Austin, Texas",
```

Comment:
**thought (nonblocking)**: above we just did "users from foo.io". for consistency maybe we make that one "Dataset of users..." or we drop "dataset of" from both?

Reply:
Yeah good point. I think "dataset" is implicit enough. We can always iterate/revisit if there's confusion! I'll make the change